pool/app/admin/admin.ml

```diff
@@ -41,3 +40,8 @@ let user_is_admin pool (user : Sihl_user.t) =
 ;;
 
 module Guard = Entity_guard
+
+module Repo = struct
+  let t = Repo_entity.t
+  let select_admins_joins_import_sql = Repo.Sql.select_admins_joins_import_sql
```

Comment:
In similar modules we used the signature

```ocaml
module Repo : sig
  module Entity : sig
    val t : t Caqti_type.t
  end

  val select_admins_joins_import_sql
    :  import_columns:string
    -> where:string
    -> limit:int
    -> string
end
```

open:
- within repo.ml: 
    ```ocaml
    module Entity = Repo_entity
    let select_admins_joins_import_sql = Sql.select_admins_joins_import_sql
    ```
- rename `select_admins_joins_import_sql` -> `select_imported_admins_sql`
- update signature in mli file

Reply:
I cannot find any occurrence of a redefinition of `Repo_entity` as `Entity` in a `repo.ml` file. That would also make `entity.ml` inaccessible. 

I defined it in `admin.ml`, like we did in the experiment module: 
```
module Repo = struct
  module Entity = struct
    include Repo_entity
  end

  let select_imported_admins_sql = Repo.Sql.select_imported_admins_sql
end

```